pykokkos/lib/ufuncs.py

```diff
@@ -1,4 +1,8 @@
+from typing import Sequence
+
 import pykokkos as pk
+import numpy as np
+import math
```

Comment:
Two things: move `import math` to the top, since it's a standard library import (numpy and pykokkos are 3rd party libraries), and import numpy before you import pykokkos (alphabetical order)